lib/miq_pglogical.rb

```diff
@@ -102,21 +70,10 @@ def self.refresh_excludes(new_excludes)
     pgl.refresh_excludes
   end
 
-  # Aligns the contents of the 'miq' replication set with the currently configured vmdb excludes
+  # Aligns the contents of the 'miq' publication with the currently configured excludes
   def refresh_excludes
-    pglogical.with_replication_set_lock(REPLICATION_SET_NAME) do
-      # remove newly excluded tables from replication set
-      newly_excluded_tables.each do |table|
-        _log.info("Removing #{table} from #{REPLICATION_SET_NAME} replication set")
-        pglogical.replication_set_remove_table(REPLICATION_SET_NAME, table)
-      end
-
-      # add tables to the set which are no longer excluded (or new)
-      newly_included_tables.each do |table|
-        _log.info("Adding #{table} to #{REPLICATION_SET_NAME} replication set")
-        pglogical.replication_set_add_table(REPLICATION_SET_NAME, table, true)
-      end
-    end
+    tables = ApplicationRecord.connection.tables - configured_excludes
```

Comment:
I could see moving excludes into the client as well, as a feature, but that's probably a bit more contentious and would require discussion.

Reply:
Yeah, I'm not so sure about this one. The existing API is around included tables so if anything I would like to continue that on if I make a change to the client gem.

But if I go that route then I'm not sure how much I could really improve what's already there as `create_publication` already takes and array of tables to include.